README.md

```diff
@@ -239,11 +239,90 @@ class BackportPublishedDefaultOnPosts < ActiveRecord::Migration
 end
 ```
 
+### Renaming columns
+
+#### Bad
+
+Renaming columns is unsafe!
+
+This action can will make existing running code brake which depend on the column.
```

Comment:
Little typo, should be "break" instead of "brake"

```suggestion
This action can will make existing running code break which depend on the column.
```